packages-answers/utils/src/findSidekickById.ts

```diff
@@ -84,8 +85,11 @@ export async function findSidekickById(user: User, id: string) {
         .flat()
 
     const missingCredentials = extractMissingCredentials(chatflow.flowData)
+    const allCredentials = extractAllCredentials(chatflow.flowData)
 
-    const needsSetup = missingCredentials.length > 0
+    // In QuickSetup mode, show all credentials; otherwise show only missing ones
+    const credentialsToShow = isQuickSetup ? allCredentials : missingCredentials
+    const needsSetup = isQuickSetup ? allCredentials.length > 0 : missingCredentials.length > 0
```

Comment:
Lets make sure extractAllCredentials returns the missing ones marked as "missing"